data/free worlds intro.txt

```diff
@@ -919,15 +919,15 @@ mission "FW Katya 4"
 				`	"Was anyone killed?"`
 				`	"How did you escape?"`
 			
-			`	"The eruption was on the other continent," she explains, "so no one was killed in the blast. It was a slow-moving disaster. We had six months or so to move two million settlers off-world before the temperatures would drop too low to be livable. More than half of the Navy was mobilized to ferry our people to other planets."`
+			`	"The eruption was on another continent," she explains, "so no one was killed in the blast. It was a slow-moving disaster. We had six months or so to move two million settlers off-world before the temperatures dropped too low to be livable. More than half of the Navy was mobilized to ferry our people to other planets."`
```

Comment:
>unless the planet only had two continents

How do you know it doesn't? Earth has had two continents at times, even one. It's even hypothesized that when Earth had only one or two continents that were located in just the right position, this helped to cause a global ice age (Snowball Earth theory).